README.md

```diff
@@ -2,7 +2,7 @@
 
 # Complete the deputy report
 
-This app is the [Complete the deputy report][service] service. It provides an online reporting service that has a publicly available frontend used by deputies to submit their reports, and a VPN restricted admin area for case managers to review submitted reports.
+This app is the [Complete the deputy report][service] service. It provides an online reporting service that has a publicly available frontend used by deputies to submit their reports, and a VPN restricted admin area for case managers to review submitted reports..
```

Comment:
Looks like an accidental change - sorry to be annoying! =D

Reply:
Ahhh I thought I reverted this *hands over face*